src/main/java/com/endava/Ticket_Management_System/model/TicketCategory.java

```diff
@@ -0,0 +1,69 @@
+package com.endava.Ticket_Management_System.model;
+
+import jakarta.persistence.*;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+@Table(name = "TicketCategory")
+public class TicketCategory implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int ticketCategoryID;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "price")
+    private BigDecimal price;
+
+    @ManyToOne
+    @JoinColumn(name = "eventID")
+    private Event event;
+
+//    @OneToMany(mappedBy = "TicketCategory",cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+//    private List<Orders> ordersList;
+
+    public TicketCategory() {
```

Comment:
Why didn't you use lombok for constructors, getter and setters? I see you have it as a dependency